docs/concepts/public-utilities.md

```diff
@@ -34,6 +34,29 @@ From there, an internal system extracts the relevant information from the reques
 
 Your browser may have a local cache of the content in question and might not reflect that something has been blocked on the gateways. To avoid browser caching, attempt to view the content using your browser's incognito or private mode. You can also prevent caching issues by using a command-line tool such as Curl or Wget.
 
+### Supported Features
+
+IPFS implementations can support a variety of protocols and specifications. The above public gateways support the following:
+
+- ipfs.io and dweb.link support the full set of IPFS Gateway specifications: https://specs.ipfs.tech/http-gateways/
+- trustless-gateway.link supports only the Trustless Gateway subset: https://specs.ipfs.tech/http-gateways/trustless-gateway/
+
+They support HTTP clients reaching them over both ipv4 and ipv6 addresses
+
+The underlying IPFS nodes backing the gateways support the following mutable identifiers under the `/ipns` namespace:
+- IPNS Public Keys
+- DNSLink for all IANA registered domains as well as .crypto and .eth
+
+The underlying IPFS nodes backing the gateways support retrieving data from peers that:
+- Have either ipv4 or ipv6 addresses
+- Are either reachable over the public internet or are accessible via libp2p's relay-v2 protocol and reach out to the gateway nodes via dialback
+- Support one of the following libp2p transport configurations:
+   - QUIC-v1
+   - TCP or WS or WSS, Yamux, TLS or Noise
+   - WebTransport
```

Comment:
WebTransport is only for browsers as far as I'm aware. 
```suggestion
```

Reply:
go-libp2p should support both receiving and dialing webtransport addresses (e.g. https://github.com/libp2p/go-libp2p/blob/6cebdd88366696a19b61dc209768c932a8f1ec4b/p2p/transport/webtransport/transport_test.go#L106)

Reply:
Pretty cool, but would be misleading. As far as I'm aware there's no reason a go peer would use WebTransport over QUIC. It would be misleading as it might wrongly suggest that browsers with WebTransport can be dialled over WebTransport.

Reply:
> As far as I'm aware there's no reason a go peer would use WebTransport over QUIC

IIRC WebTransport would better help you hide you were using libp2p since it just looks like a regular HTTP/3 connection. However, independent of that some environments other than browsers might make WebTransport available before QUIC due to needing lower level access to QUIC (as opposed to WebTransport which for libp2p's case runs a Noise handshake on top to establish which PeerID is associated with the connection).